src/biogeophys/SoilStateInitTimeConstMod.F90

```diff
@@ -702,7 +703,8 @@ subroutine SoilStateInitTimeConst(bounds, soilstate_inst, nlfilename)
     do c = begc,endc
        g = col%gridcell(c)
 
-       soilstate_inst%gwc_thr_col(c) = 0.17_r8 + 0.14_r8 * clay3d(g,1) * 0.01_r8
+       soilstate_inst%gwc_thr_col(c) = 0.01_r8*(0.17_r8*clay3d(g,1) + 0.0014_r8*clay3d(g,1)*clay3d(g,1))  ! Danny M. Leung modified the equation of soil moisture effect for dust emissions using a scale factor of 1. 0.01 is to convert from % to fraction.
```

Comment:
This comment should be changed to be more general or removed...

Reply:
This one is for Zender. The above one is for Kok. I do prefer using this one for both schemes in the main. I plan to comment the above line for Kok, and add a comment to explain what it is.